index.html

```diff
@@ -1,65 +1,247 @@
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+
+                        <li class="project">
+                            <h3>Profesional Art<br>Printing Data</h3>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                            <ul class="build-with">
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                            <button type="button">See Project</button>
+                        </li>
+                    </ul>
+                </div>
+            </section>
+        
+            <!--About Me Section-->
+            <section class="about">
+                <div class="about-me">
+                <h2>About me</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button type="button" class="btn">Get My Resume</button>
+                </div>
+                <hr>
```

Comment:
- [ ]  kindly add the margins between the line and the background to match the Figma design


![image](https://user-images.githubusercontent.com/89282221/185761640-ad34f343-4a9f-4a1c-9993-368c0a4dd238.png)